pkg/runner/run_context.go

```diff
@@ -584,10 +584,9 @@ func withDefaultBranch(b string, event map[string]interface{}) map[string]interf
 func (rc *RunContext) withGithubEnv(env map[string]string) map[string]string {
 	github := rc.getGithubContext()
 	env["CI"] = "true"
-	env["HOME"] = "/github/home"
-
-	env["GITHUB_ENV"] = "/github/workflow/envs.txt"
-  env["GITHUB_PATH"] = "/github/workflow/paths.txt"
+	env["HOME"] = fmt.Sprintf("%s/%s", filepath.Dir(rc.Config.Workdir), "home")
+	env["GITHUB_ENV"] = fmt.Sprintf("%s/%s", filepath.Dir(rc.Config.Workdir), "workflow/envs.txt")
+	env["GITHUB_ENV"] = fmt.Sprintf("%s/%s", filepath.Dir(rc.Config.Workdir), "workflow/paths.txt")
```

Comment:
```suggestion
	env["GITHUB_PATH"] = fmt.Sprintf("%s/%s", filepath.Dir(rc.Config.Workdir), "workflow/paths.txt")
```

Reply:
Yes, I've fixed it now.

But I just figured out that the test I wrote for setting the PATH from GITHUB_PATH doesn't work anymore (I think it did work earlier)

When I fetch $PATH inside a step in my test it's always `/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin`, so it doesn't seem to be updated even though the envList that is passed to `ContainerExecCreate` in `docker_run.go` contains the updated PATH.
This might be that way `ContainerExecCreate` has worked all along, but this test run worked for instance https://github.com/nektos/act/actions/runs/675631386

Reply:
I've been looking into ways to update the PATH of a container before a ContainerExecCreate, but maybe the only way around this is to prepend the command with a  `fmt.Sprintf("export PATH=%s", env["PATH"])`

Reply:
Wouldn't that break on different shells (`zsh`, `pwsh`)?

Reply:
I'm suspecting that #570 breaks `PATH`, currently testing it on your branch

Reply:
Probably, so one would need to set the PATH env in a switch on `s.Shell` like in [workflow.go](https://github.com/nektos/act/blob/master/pkg/model/workflow.go#L257)

Did a quick test now locally with `run = fmt.Sprintf("export PATH=%s; %s", sc.Env["UPDATED_PATH"], run)` in `step_context.go` and that seems to work. 
So now I just have to figure out how to set the PATH env for the different shells returned by `ShellCommand()`

Reply:
> I'm suspecting that #570 breaks `PATH`, currently testing it on your branch

Aha, that's something that has changed since my first commit. 
I can confirm that swapping back to `bash --noprofile --norc -e {0}` does make my test pass

Reply:
Yep, tried it now and it works for me too. I would be in favour of reverting #570 if there would be any other way to source `/etc/environment`

Reply:
Also a heads up, there is another PR that touches shells => #575

Reply:
I can see if I get some spare time over Easter to look at the `/etc/environment` merging with `env`